tests/tests.bats

```diff
@@ -46,8 +46,8 @@ SUT_IMAGE=$(get_sut_image)
   cd "${BATS_TEST_DIRNAME}"/.. || false
 
   local ARG_TEST_VERSION
-  local TEST_VERSION="4.3"
```

Comment:
Again, this is to test that build args like version and such can be overridden, so this should be left as is.

Reply:
I've tried the following: https://github.com/jenkinsci/docker-inbound-agent/pull/286